secrets-provider-chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{/*
+Return the most recent RBAC API available
+*/}}
+{{- define "conjur.rbac-api" -}}
+{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
+{{- printf "rbac.authorization.k8s.io/v1" -}}
+{{- else if .Capabilities.APIVersions.Has "v1" }}
+{{- printf "v1" -}}
+{{- end }}
+{{- end }}
```

Comment:
i am not familiar with helm. should this be indented?

Reply:
She Helm's GH repo [here](https://github.com/helm/charts/blob/b05e749688f2c53f76cc5cdcdc06c02acdf24aaf/stable/telegraf/templates/_helpers.tpl#L14)
Only very specific things are indented (range, etc)
I would imagine because every indent is shown once rendered and yaml is indent sensitive